src/CommonITILObject.php

```diff
@@ -6675,16 +6675,16 @@ public function getTimelineItems(array $options = [])
             class_exists($validation_class) && $params['with_validations']
             && ($validation_class::canView() || $params['bypass_rights'])
         ) {
-            $valitation_obj   = new $validation_class();
-            $validations = $valitation_obj->find([$foreignKey => $this->getID()]);
+            $validation_obj   = new $validation_class();
+            $validations = $validation_obj->find([$foreignKey => $this->getID()]);
             foreach ($validations as $validations_id => $validation) {
-                $canedit = $valitation_obj->can($validations_id, UPDATE);
-                $cananswer = ($validation['users_id_validate'] === Session::getLoginUserID() &&
```

Comment:
check against current user was already implemented on CommonITILValidation::canValidate(). The new feature requires to check date range in a user row then replacing the comparison here by the method, as now a SQL query  is necessary.